astropy/io/fits/diff.py

```diff
@@ -262,6 +276,9 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         ignore_blank_cards : bool, optional
             Ignore all cards that are blank, i.e. they only contain
             whitespace (default: True).
+
+        tolerance : float, optional
```

Comment:
I don't think deprecated parameters should be added to the docstring. Probably best to remove it completly, the homepage contains the documentation for older versions and with the directive in `rtol` about the renaming it should be fine, right?